src/state-summary/state-card-input_number.html

```diff
@@ -86,14 +113,26 @@
       value: {
         type: Number,
       },
+      mode: {
+        type: String,
+      }
     };
   }
 
+  hiddenState() {
+    if (this.$.slider.offsetWidth < 100) {
+      this.$.sliderstate.hidden = true;
+    } else if (this.$.slider.offsetWidth >= 145) {
+      this.$.sliderstate.hidden = false;
+    }
+  }
+
   stateObjectChanged(newVal) {
     this.min = Number(newVal.attributes.min);
```

Comment:
Use setProperties to apply all properties together. This will cause less observer calls.
https://www.polymer-project.org/2.0/docs/devguide/model-data#set-property

Reply:
Improved in https://github.com/home-assistant/home-assistant-polymer/commit/00b07de52507f0f72a86edd624d7c155c960e584